colabs/scikit/decision-trees.ipynb

```diff
@@ -0,0 +1,240 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.    reg = reg.fit(x_train,y_train)
</pre><p>Don't we have any callback for sklearn's decision tree implementation? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/examples/pull/106/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
Huh, which one ? That'd make the report better as well. I can't find any references in the docs.

Reply:
I don't think we have. Currently we only cover XGBoost and LightGBM. Both of them have PRs waiting to be merged. We will soon be supporting CatBoost. Maybe we can sync and I can add a callback for sklearn as well.

Reply:
Sure sounds good. Till then let's stick to this implementation ? Thoughts ??

Reply:
Since I didn't run your colab, are you able to log any metrics? Can you share an example dashboard? As I see it only confusion matrix is logged.

Reply:
Sure Link to the workspace -> https://wandb.ai/sauravmaheshkar/Decision-Tree

Reply:
* The colab showcase the features we have currently. 
* We ain't logging any metrics since we don't have any callback. This can be a todo for future. For now I think we can merge re: point 1.